### PR TITLE
add basic form to upload pdf through api

### PR DESCRIPTION
**Pull request by u10000129**

### CODE REVIEW

File: app/controllers/pdfs.rb [hunk 1]
```diff
@@ -20,6 +21,20 @@ class App < Roda
             routing.redirect '/auth/login'
           end
         end
+
+        # POST /pdfs/
+        routing.post do
+          filename = routing.params['pdf_file'][:filename]
+          file = routing.params['pdf_file'][:tempfile]
+          content = Base64.strict_encode64(file.read)
```
Comment (nthu103060011) [inline app/controllers/pdfs.rb]: Would it be better to encode in service rather than in controller?

File: app/services/upload_pdf.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+module CoEditPDF
+  # Creates an account
+  class UploadPdf
+    class UploadError < StandardError; end
+
+    def initialize(config)
+      @config = config
+    end
+
+    def call(current_account:, filename:, file_read:)
```
Comment (nthu103060011) [inline app/services/upload_pdf.rb]: I think "content" or "file_content" would be less confusing than "file_read".

Comment (u10000129) [reply app/services/upload_pdf.rb]: fixed

File: app/controllers/pdfs.rb (hunk 1 above)
Comment (u10000129) [reply app/controllers/pdfs.rb]: fixed

**Pull request merged**